src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IDynamicInvocationExpression.vb

```diff
@@ -535,6 +536,437 @@ IDynamicInvocationOperation (OperationKind.DynamicInvocation, Type: System.Objec
 
             VerifyOperationTreeAndDiagnosticsForTest(Of InvocationExpressionSyntax)(source, expectedOperationTree, expectedDiagnostics)
         End Sub
+
+        <CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)>
+        <Fact()>
+        Public Sub DynamicInvocation_ControlFlowInReceiver()
```

Comment:
>DynamicInvocation_ControlFlowInReceiver [](start = 19, length = 39)

Are we covering scenarios of an invocation of off an object directly and no receiver case?

Reply:
Added `DynamicInvocation_ControlFlowInvocationOffObject` and `DynamicInvocation_ControlFlowNullReceiver`.

---
In reply to: [186843373](https://github.com/dotnet/roslyn/pull/26690#discussion_r186843373) [](ancestors = 186843373)